packages/core/src/Modifiers.ts

```diff
@@ -247,6 +247,22 @@ export class Modifiers extends EventMixin {
     toggle(modifier: Modifier | Constructor<Modifier>): void {
         this.remove(modifier) || this.append(modifier);
     }
+    /**
+     * Check that all `otherModifiers` ar contained within this ones.
```

Comment:
"are", "one".
Sinon, j'aurais préféré une phrase active plutôt que passive (inversée quoi).
"Return true if this array contains all of the given modifiers." ?